pyne/alara.py

```diff
@@ -16,14 +16,11 @@
 except NameError:
     basestring = str
 
-try:
-    from itaps import iMesh, iBase, iMeshExtensions
-except ImportError:
-    warn("the PyTAPS optional dependency could not be imported. "
-                  "Some aspects of the alara module may be incomplete.",
-                  QAWarning)
+from pyne.mesh import Mesh, MeshError, HAVE_PYMOAB
+
+if HAVE_PYMOAB:
+    from pyne.mesh import mesh_iterate
```

Comment:
shuoldn't a warning be printed here also in case pyMOAB is not installed?. Just to be consistent with other scripts that has the warning printed like;
```
else:
    warn("the PyMOAB optional dependency could not be imported. "
         "Some aspects of the mcnp module may be incomplete.",
         QAWarning)
```

Reply:
This was resolved in #52